### DISCUSSION TITLE
Unable to replicate my neovim theme

### DISCUSSION THREAD

**Discussion by siduck** [PrismJS/prism#3469] [Q&A]
Hi guys im porting my neovim theme to prism but the colors dont match,  Im unable to replicate it 

look 

![image](https://user-images.githubusercontent.com/59060246/173262636-f7ca7260-6d8d-453b-a55e-31b24280310a.png)

I tried highlighting .token operator but it then highlights other classes such as .token plain, .token property-access. How do i fix this :( 

For example : 

https://user-images.githubusercontent.com/59060246/173263816-9ad11388-a5a2-4b50-ba39-785b7c58da3f.mp4

nvm I give up

**Comment by RunDevelopment** [comment]
Looks like you have another Prism theme loaded aside from the one you're working on. The `color` property not doing anything while the `background` is working means that something is overriding `color`, and I suspect it's another theme.
